client/my-sites/customer-home/cards/features/stats-v2/index.jsx

```diff
@@ -0,0 +1,265 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { connect, useDispatch } from 'react-redux';
+import { numberFormat, useTranslate } from 'i18n-calypso';
+import { Card } from '@automattic/components';
+import { times } from 'lodash';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import Chart from 'components/chart';
+import Spinner from 'components/spinner';
+import QuerySiteStats from 'components/data/query-site-stats';
+import InlineSupportLink from 'components/inline-support-link';
+import { localizeUrl } from 'lib/i18n-utils';
+import { buildChartData } from 'my-sites/stats/stats-chart-tabs/utility';
+import isUnlaunchedSite from 'state/selectors/is-unlaunched-site';
+import { getSiteOption } from 'state/sites/selectors';
+import { requestChartCounts } from 'state/stats/chart-tabs/actions';
+import { getCountRecords } from 'state/stats/chart-tabs/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import {
+	getMostPopularDatetime,
+	getTopPostAndPage,
+	getViewAndVisitors,
+	isLoadingStats,
+} from './utils';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const placeholderChartData = times( 7, () => ( {
+	value: Math.random(),
+} ) );
+
+export const StatsV2 = ( {
+	chartData,
+	chartQuery,
+	insightsQuery,
+	isLoading,
+	isSiteUnlaunched,
+	mostPopularDay,
+	mostPopularTime,
+	siteId,
+	siteSlug,
+	topPage,
+	topPost,
+	topPostsQuery,
+	views,
+	visitors,
+	visitsQuery,
+} ) => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const showTopPost = !! topPost;
+	const showTopPage = !! topPage;
+	const showViews = ! showTopPost || ! showTopPage;
+	const showVisitors = ! showTopPost && ! showTopPage;
+
+	useEffect( () => {
+		if ( ! isSiteUnlaunched ) {
+			dispatch( requestChartCounts( chartQuery ) );
+		}
+	}, [ isSiteUnlaunched ] );
+
+	return (
+		<div className="stats-v2">
+			{ ! isSiteUnlaunched && (
+				<>
+					<QuerySiteStats siteId={ siteId } statType="statsVisits" query={ visitsQuery } />
+					<QuerySiteStats siteId={ siteId } statType="statsInsights" query={ insightsQuery } />
+					<QuerySiteStats siteId={ siteId } statType="statsTopPosts" query={ topPostsQuery } />
+				</>
+			) }
+			<h2 className="stats-v2__heading customer-home__section-heading">
+				{ translate( 'Stats at a glance' ) }
+			</h2>
+			<Card>
+				{ isSiteUnlaunched && (
+					<Chart data={ placeholderChartData } isPlaceholder>
+						<div>
+							{ translate( 'Launch your site to see a snapshot of traffic and insights.' ) }&nbsp;
```

Comment:
nbsp?

Reply:
Intended for separating text from link. Might try adding a CSS tweak to get rid of this.

Reply:
https://github.com/Automattic/wp-calypso/pull/41441/commits/a9c6071795f14f39b4badbffe094a4c3289466a7